### ISSUE TITLE
Not working on Chrome 83 | macOS 10.13.6

### ISSUE THREAD

**Issue by bySabi** [jkanchelov/pixi-typescript-boilerplate#115]
I can't get it to work on Chrome 83.

It gives me this error:

```
Renderer.js: 50 Uncaught Error: WebGL unsupported in this browser, use "pixi.js-legacy" for fallback canvas2d support.
    at Function.create (Renderer.js: 50)
    at autoDetectRenderer (autoDetectRenderer.js: 36)
    at new Application (Application.js: 67)
    at Main.createRenderer (index.ts: 40)
    at Main.window.onload (index.ts: 15)
create @ Renderer.js: 50
autoDetectRenderer @ autoDetectRenderer.js: 36
Application @ Application.js: 67
createRenderer @ index.ts: 40
Main.window.onload @ index.ts: 15
load (async)
Main @ index.ts: 14
(anonymous) @ index.ts: 74
parcelRequire.index.ts.pixi.js @ index.ts: 75
newRequire @ src.77de5100.js: 47
(anonymous) @ src.77de5100.js: 81
(anonymous) @ src.77de5100.js: 120
service-worker.js: 10 Attempting to install service worker and cache static assets
(index): 31 ServiceWorker registration successful with scope: http: // localhost: 63647 /
service-worker.js: 44 Activating new service worker ...

```

I have checked the Chrome WebGL support and it is working correctly and in Firefox the example works.
 I played PixiJS for the first time today and I can't imagine where the bug is.

**Comment by bySabi** [comment]
it seems a problem on PixiJS: https://github.com/pixijs/pixi.js/issues/5694

**Comment by jkanchelov** [comment]
@bySabi see if your hardware acceleration is turned on ( it is on by default, but maybe it is changed). another way is to use pixi.js-legacy package so you can have that canvas fallback support which is not currently integrated

**Comment by bySabi** [comment]
Hi @jkanchelov  thanks for this repo, it works great and is upto date.

I have reviewed the settings in Chrome and it seems that everything is correct. I also tried the nvidia webdrivers and there was no luck either.

It happens to a lot of people and they have been working on a solution since last year. Yesterday they marked the issue as a high priority and added it to their bounty bug program: https://github.com/pixijs/pixi.js/issues/5902

At the moment I will develop only with Firefox and wait for a new release.
